### PR TITLE
Add code resources page

### PR DESCRIPTION
**Pull request by danielnordh**
PR working on #1024 

Add information about the various community related projects that includes code.

- Bitcoin UI Kit packages
- Wallets based on the reference designs

### CODE REVIEW

**Comment by danielnordh** [comment]
Finalised descriptive texts and links to projects and resources.
Moving out of draft

File: guide/resources/code-resources.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+-->
+
+# Code resources
+
+Although mostly focused on design, some members of the community also have their hands in various code resources that are based on, or influenced by the Bitcoin Design Guide. They are all open source and free to use, modify and contribute to.
+
+## Bitcoin UI Kit packages
+
+The Bitcoin UI Kit is a great resource for designing bitcoin products. While the main effort is on the design components and Figma files, there are a couple of code implementations by community memebers that make adopting the designs in the guide much easier.
```
Comment (GBKS) [inline guide/resources/code-resources.md:25]:
```suggestion
The [Bitcoin UI Kit](https://www.bitcoinuikit.com) is a great resource for designing bitcoin products. While the main effort is on the design components and design source files, there are a couple of code implementations by community members that make adopting the designs in the guide much easier.
```
Adding a link for convenience, type, and removing Figma to be more general (ahem, Penpot, ahem).

File: guide/resources/code-resources.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: guide
+title: Code resources
+description: Bitcoin Design Guide code resources.
```
Comment (GBKS) [inline guide/resources/code-resources.md:4]:
```suggestion
description: Code implementations of reference designs and other resources from the Bitcoin Design Guide, created by the community.
```
Making the description more elaborate.

File: guide/resources/code-resources.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Code resources
+
+Although mostly focused on design, some members of the community also have their hands in various code resources that are based on, or influenced by the Bitcoin Design Guide. They are all open source and free to use, modify and contribute to.
+
+## Bitcoin UI Kit packages
+
+The Bitcoin UI Kit is a great resource for designing bitcoin products. While the main effort is on the design components and Figma files, there are a couple of code implementations by community memebers that make adopting the designs in the guide much easier.
+
+- Bitcoin UI for [Swift](https://github.com/reez/WalletUI)
+- Bitcoin UI for [Flutter](https://pub.dev/packages/bitcoin_ui_kit)
```
Comment (GBKS) [inline guide/resources/code-resources.md:28]: This links to a page that says "discontinued".

File: guide/resources/code-resources.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+- Bitcoin UI for [React Native](https://github.com/EttaWallet/etta-ui) (Etta UI)
+
+## Wallet implementations
+
+Several different projects are building on the reference designs from the guide. They all have slightly different goals and are at different stages, but are all fantastic learning resources for other people wanting to build a wallet.
+
+### Etta wallet
+
+[Etta Wallet](https://github.com/EttaWallet/EttaWallet) is a resource for experimenting with non-custodial lightning with a strong bias towards usability, accessibility and good UX. Built with React Native.
+The project is strongly inspired by the [daily spending wallet]('/guide/daily-spending-wallet/').
```
Comment (GBKS) [inline guide/resources/code-resources.md:38]:
```suggestion
The project is strongly inspired by the [daily spending wallet]({{ '/guide/daily-spending-wallet/' | relative_url }}).
```

**Review by GBKS — CHANGES_REQUESTED** [review]
Looking good. Had some minor feedback, and please also make sure to add an entry on the [resources landing page](https://deploy-preview-1027--bitcoin-design-site.netlify.app/guide/resources/), and update the next/back buttons in the "Design challenges" and "Contribute to guide" pages.

Have you looked for cross-linking opportunities in other pages to link to this new one?

**Comment by danielnordh** [comment]
Todo:

- Add Icon packages
- Add links to How-to guides written by Daniel?

File: guide/resources/code-resources.md (hunk 3 above)
Comment (danielnordh) [reply guide/resources/code-resources.md:28]: Fixed (this stemmed from changing the name of the package)

**Comment by danielnordh** [comment]
Addressed feedback.
Added Bitcoin Icons and Guides.
Added crosslink from Daily Wallet reference page.

**Review by GBKS — APPROVED** [review]
ACK. Looking good, and a nice addition that should make these libraries more discoverable and I hope will keep growing over time.

**Pull request merged**